gtk/docs.md

```diff
@@ -353,3 +353,13 @@ See also the `Widget:tooltip-markup` property and
 `Tooltip::set_markup`.
 ## `markup`
 the contents of the tooltip for `self`, or `None`
+<!-- impl FileChooserDialog::fn with_buttons -->
+This function creates a `FileChooserDialog` with buttons:
+```no_run
+let dialog = FileChooserDialog::with_buttons::<Window>(
+  Some("Open File"),
```

Comment:
Could you add two more whitespaces in order to fix the indent for each line please?